base-isa.md

```diff
@@ -22,48 +23,45 @@ The highest two bits of the first byte are a format marker:
 | First byte    | Second Byte  | Comment                                  |
 |:--------------|:-------------|:-----------------------------------------|
 | `00 01 CCCC`  | `RRR RRR 00` | 2 register computation                   |
-| `00 SS CCCC`  | `RRR RRR ??` | when `SS != 01`, reserved for extensions |
-| `00 01 CCCC`  | `RRR RRR ??` | when `?? != 00`, reserved for extensions |
-| `00 01 11??`  | `RRR RRR ??` | reserved for extensions                  |
+| `00 01 11??`  | `???? ????`  | reserved for extensions                  |
+| `00 SS CCCC`  | `RRR RRR MM` | when `MM != 00`, reserved for extensions |
 | `01 01 CCCC`  | `RRR IIIII`  | immediate and 1 register computation     |
-| `01 SS CCCC`  | `RRR IIIII`  | when `SS != 01`, reserved for extensions |
+| `0? SS CCCC`  | `???? ????`  | when `SS != 01`, reserved for extensions |
 | `10 0 D CCCC` | `DDDDDDDD`   | (conditional) relative jump instruction  |
 | `10 1 ?????`  | `?????????`  | reserved for extension                   |
 | `11 ??????`   | `?????????`  | reserved for extensions                  |
 
-| Symbol | Meaning                      |
-|--------|------------------------------|
-| C      | opcode bit                   |
-| R      | register id                  |
-| I      | immediate                    |
-| D      | displacement                 |
-| S      | size (res.d)                 |
-| ?      | reserved for unknown purpose |
+| Symbol | Meaning         |
+|--------|-----------------|
+| C      | Opcode          |
+| R      | Register Id     |
+| I      | Immediate       |
+| D      | Displacement    |
+| S      | Operation Size  |
+| M      | Memory Mode     |
+| ?      | Arbitrary Value |
 
```

Comment:
```suggestion

Notice that whenever `MM` bits are present, it is reserved for their value to be anything other than `00`. Similarly, it is reserved for `SS` bits to have values other than `01`.

```

For emphasis, since I don't think _any_ way of writing that table is going to be perfectly clear.